tests/drop_counters/test_drop_counters.py

```diff
@@ -85,13 +86,19 @@ def setup(duthost, testbed):
     for dut_port, neigh in mg_facts['minigraph_neighbors'].items():
         neighbor_sniff_ports.append(mg_facts['minigraph_port_indices'][dut_port])
 
+    for vlan_name in mg_facts["minigraph_vlans"].keys():
+        match = re.findall("\d{1,4}", vlan_name)
```

Comment:
{1,4} is hard to read and assumes vlan name must include vlan id. Can you use vlanid field?